.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: java
 
 install: ./gradlew assemble
-script: ./gradlew check
+script: ./gradlew check -i
```

Comment:
We should remove the `-i` again I suppose.

Reply:
I'm fine leaving it in, but it should be `--info`.

Reply:
I strongly suggest not to remove it - the only way to see the error output on travis is from the console - I don't think it's correct to add `--info` back and rerun every time we need to diagnose. @lptr is right, we should make it `--info`.

Reply:
Don't we publish a build scan from travis? We should just use it to see the error output.